packages/business-rules/src/config.js

```diff
@@ -175,12 +175,12 @@ const config = {
 						},
 						reference: appeal.id
 					}),
-					saveAndReturnContinueAppeal: (appeal, baseUrl, deadlineDate) => ({
+					saveAndReturnContinueAppeal: (appeal, encryptedId, baseUrl, deadlineDate) => ({
 						recipientEmail: appeal.email,
 						variables: {
 							'application number': appeal.planningApplicationNumber,
 							date: format(deadlineDate, 'dd MMMM yyyy'),
-							link: `${baseUrl}/full-appeal/submit-appeal/enter-code`
+							link: `${baseUrl}/appeal-householder-decision/enter-code/${encryptedId}`
```

Comment:
This appears to be using the same URL structure as the householder appeal above. Is that correct for this appeal type?